packages/front/src/routes/search/[name]/Search.controller.ts

```diff
@@ -5,13 +5,15 @@ import { returnInfinitePromise } from "$lib/helpers/promiseHelper";
 import { decodeQuerySearch, encodeQuerySearch } from "$lib/helpers/urlHelper";
 import { isRna, isSiren, isSiret } from "$lib/helpers/identifierHelper";
 import associationService from "$lib/resources/associations/association.service";
+import { isAssociation } from "$lib/resources/associations/association.helper";
 
 export default class SearchController {
     inputSearch: Store<string | undefined>;
     associations = new Store<PaginatedAssociationNameDto>({ nbPages: 1, page: 1, total: 0, results: [] });
     searchPromise: Store<Promise<unknown>>;
     duplicatesFromIdentifier: Store<string[] | null>;
     currentPage = new Store(1);
+    lastSearchCompany = new Store(false);
```

Comment:
```suggestion
    isLastSearchACompany = new Store(false);
```

Je trouve les noms de variables booléennes en `is` ou `has ` + clairs. Si non on s'attend à la dernière entreprise cherchée dans la variable